foundation/foundation-upgrade/src/test/resources/specifications/v1_7_0/data-lineage-migration.feature

```diff
@@ -0,0 +1,11 @@
+Feature: Data Lineage upgrade migration
+
+  Scenario: Migrate the Data Lineage module files with the new package name when the VERSION_AISSEMBLE is less than the 1.7.0.
+    Given an existing downstream project with data lineage is less than VERSION_AISSEMBLE_one_seven_zero
+    When the data lineage version migration executes
+    Then the imports are updated in the data lineage modules with the new package name
+
+  Scenario: Do not migrate the Data Lineage module files with the new package name when the imports are already correct
```

Comment:
A: I'm not sure this is necessary.  At least if you rewrite the first scenario as recommended, I think it becomes clear that any file _not_ referencing the old package name won't be migrated.